### DISCUSSION TITLE
Extension of the color pallet?

### DISCUSSION THREAD

**Discussion by deku-m** [netbox-community/netbox#13213] [Ideas]
Hi,

I was wondering if the color pallet can be extended to the default 140 colors supported by all (modern) browsers as named on:
https://www.w3schools.com/colors/colors_names.asp

Gives us more colors to choose from and supported.
Or even the option to use hex codes perhaps?

**Comment by jeremystretch** [comment]
The color palette in use is intentionally limited to ensure that colors can be readily distinguished from one another.

**Reply by deku-m** [reply]
Well some point could be that you run out of colors as now its 27. 
Maybe a solution is that you it is possible to add or use the #hexcode to use for the color you want?

**Reply by jeremystretch** [reply]
The solution is to reuse a color, or abandon the use of color as a visual aid. A user cannot easily distinguish between e.g. "black" and "slightly darker black" so it's pointless to differentiate to that degree.

**Comment by gdprdatasubect** [comment]
Hex-Codes are already possible when importing things, e.g. Cables
